README.md

```diff
@@ -165,5 +250,13 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+genie_lamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name:"Genie",lamp: genie_lamp)
+genie_lamp.wishes_remaining = 1
+   would you need a genie_lamp.save? I assume that since it's a class method it is not needed
```

Comment:
yes you would need to call `.save` here on the instance after you use the setter method, ie.:
```rb
genie_lamp.wishes_remaining = 1
genie_lamp.save
```
 Otherwise you can do it in one line with an `.update` like: `genie_lamp.update(wishes_remaining: 1)`